### DISCUSSION TITLE
unrecognized arguments: --always-normal-vram

### DISCUSSION THREAD

**Discussion by rionix88** [fenneishi/Fooocus-Control#24] [Q&A]
after update to latest version in stability matrix, i cannot run with normal vram. only show error

usage: launch.py [-h] [--listen [IP]] [--port PORT]
                 [--enable-cors-header [ORIGIN]]
                 [--extra-model-paths-config PATH [PATH ...]]
                 [--output-directory OUTPUT_DIRECTORY]
                 [--temp-directory TEMP_DIRECTORY]
                 [--input-directory INPUT_DIRECTORY] [--auto-launch]
                 [--disable-auto-launch] [--cuda-device DEVICE_ID]
                 [--cuda-malloc | --disable-cuda-malloc]
                 [--dont-upcast-attention] [--force-fp32 | --force-fp16]
                 [--bf16-unet] [--fp16-vae | --fp32-vae | --bf16-vae]
                 [--directml [DIRECTML_DEVICE]] [--disable-ipex-optimize]
                 [--preview-method [none,auto,latent2rgb,taesd]]
                 [--use-split-cross-attention | --use-quad-cross-attention | --use-pytorch-cross-attention]
                 [--disable-xformers]
                 [--gpu-only | --highvram | --normalvram | --lowvram | --novram | --cpu]
                 [--disable-smart-memory] [--dont-print-server]
                 [--quick-test-for-ci] [--windows-standalone-build]
                 [--disable-metadata] [--share] [--preset PRESET]
launch.py: error: unrecognized arguments: --always-normal-vram

Process finished with exit code 2

**Comment by aserechenko** [comment]
Same problem.
This argument exists in original Fooocus: https://github.com/lllyasviel/Fooocus/blame/6da0441cc75d2e40df76780109a2ea4bcb58c1c1/readme.md#L376
But not in this repo.
If authors could sync this fork, then the problem will resolve.
